### PR TITLE
Replace all mentions of `build cop` with `build on-duty`

### PR DESCRIPTION
**Pull request by rsimha**
This is something we've been meaning to do for a while.

**Companion PR:** https://github.com/ampproject/amphtml/pull/33465

**Next steps:**

- [ ] Merge and deploy this PR
- [ ] Update the rotation tool to reflect this name
- [ ] Update the name of the GitHub team
- [ ] Merge https://github.com/ampproject/amphtml/pull/33465

### CODE REVIEW

File: onduty/redacted.env [hunk 1]
```diff
@@ -2,5 +2,5 @@
 GITHUB_ACCESS_TOKEN=[REDACTED]
 GITHUB_ORG=ampproject
 RELEASE_ONDUTY_TEAM=release-on-duty
-BUILD_COP_TEAM=build-cop
+BUILD_ON_DUTY_TEAM=build-on-duty
```
Comment (danielrozenberg) [inline onduty/redacted.env:5]: ONDUTY/ON_DUTY >_<

**Review by danielrozenberg — APPROVED** [review]
build caretaker?
build monitor?
build hall monitor?

File: onduty/redacted.env (hunk 1 above)
Comment (rsimha) [reply onduty/redacted.env]: Nice catch. Fixed.

Comment (danielrozenberg) [reply onduty/redacted.env]: I would suggest the opposite, replace `ONDUTY` with `ON_DUTY`. Also, don't forget to change it in both the env file and in the code!

Comment (rsimha) [reply onduty/redacted.env:5]: Agree. Done, and cross-checked.

**Comment by rsimha** [comment]
FYI @ampproject/build-on-duty

**Comment by rsimha** [comment]
@rcebulko @danielrozenberg I've deployed a new version with this commit, but do not have permissions to see if the `amp-onduty-bot` app on Google Cloud was correctly updated and the renamed (non-redacted) env vars were correctly set. Could I ask one of you to check at leisure? (There's no hurry since the next time this app needs to update the team (now renamed to `@ampproject/build-on-duty` is next week.)

**Comment by rcebulko** [comment]
Looks fine to me

**Pull request merged**